### ISSUE TITLE
[PLUGIN] I'm worried about the war plugin destroying the geopolitical aspect of the server

### ISSUE THREAD

**Issue by ketchupdrinker** [EarthMC/Issue-Tracker#1166]
With the war plugin apparently coming soon to be enabled on EMC it's come to my attention that there are a lot of flaws with how the war plugin functions, especially stated in the article

Here are the parts of it that I have issues with:
**-Force disband if war fee is not paid after siege;**
It is stated in the beta article that you will be able to charge a fee of gold after besieging a town or the town will disband, the maximum on this is not specified, and if towns can be disbanded purely off of this fact then non pvp-based towns can be constantly besieged and drowned for their money until they are disbanded, which seems to break a geopolitical aspect of the game and potentially turn the server into an advanced factions.
**-Non-consensual war (both sides agreeing to war)**
The point of the war plugin was to remove 1 person nations and such, yet if there doesn't have to be a reason for war, then large pvp based nations will be able to exponentially grow with no limit on their influence, it's important to keep a power balance on the server because the entire server doesn't have a consensual agreement to being under the same side, if non consensual war is enabled it is probable that lots of players will quit due to frustrations of their nation being destroyed for no reason.
**-Monetary-based sieges**
This corresponds to sieges more than 3000 blocks away, but this could potentially cause powerful nations to disrupt a power complex and siege nations just to charge a gold fee afterwards, this allows for the large nations to continually bully new nations (it takes a lot more than a few days for a nation to develop to defend itself),
**-Sieges more than 3000 blocks away**
This also has to correspond with gold based sieges, but takes away the privilege from having an isolated nation from powerful nations, and allows large older nations with developed armies and communities to pick on newer towns and ruin the point of the server being about negotiation and geopolitics.

TL;DR
war plugin needs to be heavily modified

**(I don't have premium to check beta mode so most of this information is just from the article or from what other players said, so correct me if I'm wrong on any of these things)**

**Comment by MineHero43** [comment]
There are restrictions, for example you can’t siege a town that’s outside of your nation range and it costs in gold 2x the amount of chunks a town has to siege it. (Eg Town A has 400 chunks, it would cost 800 gold to start a siege)

The war plugin as it was on beta is perfect. Clearly you haven’t played the war plugin yourself because if you did you would know that PvP is only a small part of how sieges are won.

If you don’t want to be involved in politics and just want to build, set your town to peaceful and you won’t suffer the negative effects from being sieged, the obvious downside is you won’t be able to participate in sieges but I’m sure towns based on peaceful building won’t mind.

A consensual war plugin isn’t a war plugin, hence why it has to be non consensual. This war plugin will mainly impact one man or generally very small nations as they’d be incorporated into a larger regional power.

Frankly speaking, if you can’t defend your nation then your nation has no right to exist.

**Comment by 10op10** [comment]
War is part of Geo Politics like how dumb must you be

**Comment by remcodd2** [comment]
@MineHero43 but in peacefull you can still be captured, charged a high tax and then be disbanded right? That what i find an issue, many towns aren't very pvp and could easily be disbanded like this

**Comment by Goosius1** [comment]
Hi. One main point which needs clarification --- towns do not get deleted due to plunder or tax.

They used to, with an older version, but I developed a special bankruptcy feature about a month ago. 

So now they just go into debt (with some misc. downsides until all debts are cleared etc)

**Comment by remcodd2** [comment]
@goosis

> Hi. One main point which needs clarification --- towns do not get deleted due to plunder or tax.
> 
> They used to, with an older version, but I developed a special bankruptcy feature about a month ago.
> 
> So now they just go into debt (with some misc. downsides until all debts are cleared etc)

Good, thats an improvement in my eyes. But what kind of downsides? because that debt might be impossible to get out of if the tax is very high

**Comment by Goosius1** [comment]
Well, setup guidelines are that max tax should be set by the server to be reasonable, and not too high.

Also, consider that its not a zero-sum game for an occupier. If they tax too high, the town will be motivated to revolt and/or assist their enemies etc. Plus word will get around of this mistreatment, causing all towns to resist their attacks more strongly.

The downsides are cannot claim, unclaim, recruit or build. (not very pleasant, but a lot more pleasant than town destruction and ruin)

**Comment by FutureLemon** [comment]
> @MineHero43 but in peacefull you can still be captured, charged a high tax and then be disbanded right? That what i find an issue, many towns aren't very pvp and could easily be disbanded like this

As far as I was told and read when you're peaceful you can still be captured, but don't have to pay for the taxes.

**Comment by Goosius1** [comment]
Your sources were correct😁

**Comment by remcodd2** [comment]
> > @MineHero43 but in peacefull you can still be captured, charged a high tax and then be disbanded right? That what i find an issue, many towns aren't very pvp and could easily be disbanded like this
> 
> As far as I was told and read when you're peaceful you can still be captured, but don't have to pay for the taxes.

Ah that seems like a fair trade off. Still captured while peacefull but no taxes. Could a developer confirm this?

**Comment by 10op10** [comment]
> > > @MineHero43 but in peacefull you can still be captured, charged a high tax and then be disbanded right? That what i find an issue, many towns aren't very pvp and could easily be disbanded like this
> > 
> > 
> > As far as I was told and read when you're peaceful you can still be captured, but don't have to pay for the taxes.
> 
> Ah that seems like a fair trade off. Still captured while peacefull but no taxes. Could a developer confirm this?

He just did

**Comment by remcodd2** [comment]
ye im blind already deleted the comment

**Comment by MineHero43** [comment]
> > @MineHero43 but in peacefull you can still be captured, charged a high tax and then be disbanded right? That what i find an issue, many towns aren't very pvp and could easily be disbanded like this
> 
> As far as I was told and read when you're peaceful you can still be captured, but don't have to pay for the taxes.

Yeah exactly, you can still be captured but you don't suffer the negative effects

**Comment by Scorpionzzx** [comment]
War plugin is very balanced and well-done ngl. The plugin also makes it economically bad to siege because of the high costs. Its likely even the richest nations will be very conservative and precise with how they spend it and definitely won't "destroy the geopolitical aspect of the server. In fact war plugin will only increase things like diplomacy because of the possible loss of gold for BOTH sides. Personally I think the only thing needed is more clarification about the taxes/disbanding mechanics as this is what scares people the most and it should be better explained/updated to the public so people don't think there towns and stuff will get disbanded or they will get taxed 100g a day or something.Also, one benefit not talked about is the war plugin also having  keep inventory on which basically means fewer god sets will be lost in battles. I also agree with Minehero43's points above^

**Comment by ketchupdrinker** [comment]
@MineHero43 I've had some people tell me in the past that you can siege towns over 3000 chunks away but, it doesn't let you add them to your nation, you can just charge them taxes and disband their nation. Do you know if that's true?

**Comment by MineHero43** [comment]
> @MineHero43 I've had some people tell me in the past that you can siege towns over 3000 chunks away but, it doesn't let you add them to your nation, you can just charge them taxes and disband their nation. Do you know if that's true?

Karl mentioned in a previous github that you'll only be able to siege towns within your nation range

**Comment by unbatedaura409** [comment]
If you played with the war plugin and have actually besieged a town, you'd know pvp isn't a large part of it. Most of it involves accumulating points to get the upper leg on the besieged, and at the end of 3 days whoever has the most points wins the siege. Pls do more research next time

**Comment by Goosius1** [comment]
Although that was the case on EMC Beta, remember that most of the time there were less than 10 people online at any one time.

Thus, opposing soldiers (who might otherwise do combat) were often just not online at the same time.

The experience of large servers (with many online players) who use the system is quite different. If the system is enabled, expect occasional massive PVP battles if wars break out.
